resources/function_help/json/geometry_overlay_contains

```diff
@@ -27,14 +37,10 @@
       "arg": "cache",
       "description": "set this to true to build a local spatial index (most of the time, this is unwanted, unless you are working with a particularly slow data provider)",
       "optional": true,
-      "default": false
+      "default": "false"
```

Comment:
Is "default" different from "optional" (you quoted `"false"` but not `true` one line above)

Reply:
`Optional` is whether the argument can be ignored or not so I guess it expects a Boolean (and as is, it renders correctly in GUI). `Default` is the value it takes if not used, and looking to the current GUI, it's not displayed, while quoting the value makes it appear in the dialog (as you can see above). I don't know what happens if not quoted.
Sorry. I can explain only based on experience (the behavior I see in QGIS), not code wise.